addDestinationProcess.php

```diff
@@ -0,0 +1,80 @@
+<?php
+require 'connection.php';
+
+$name;
+$description;
+$categories = null;
+$itemList = null;
+$imageFile = null;
+
+$jsonText = $_POST["otherDetails"];
```

Comment:
You must validate the value before assigning it to the variable.

Reply:
ok